service/src/main/frontend/src/env.d.ts

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable */
+
+declare namespace NodeJS {
+  interface ProcessEnv {
+    NODE_ENV: string;
+    VUE_ROUTER_MODE: 'hash' | 'history' | 'abstract' | undefined;
+    VUE_ROUTER_BASE: string | undefined;
```

Comment:
Probably don't need these router settings if we exclude vue routing.

Reply:
Weird, when I remove these settings, it doesn't affect anything but removing the index.ts does. I tried to get rid of it in the quasar.config.js file but it still persists to break it when removing /router